Arrowgene.Ddon.GameServer/Handler/ItemSellItemHandler.cs

```diff
@@ -26,8 +26,11 @@ public override void Handle(GameClient client, StructurePacket<C2SItemSellItemRe
 
             uint totalAmountToAdd = 0;
 
-            S2CItemUpdateCharacterItemNtc ntc = new S2CItemUpdateCharacterItemNtc();
-            ntc.UpdateType = 267;
+            S2CItemUpdateCharacterItemNtc ntc = new S2CItemUpdateCharacterItemNtc()
+            {
+                UpdateType = (ItemNoticeType)267 // No named enum for this?
```

Comment:
Add the value to the named enum, you can make up a name, SellItem for example is fine

Reply:
One thing I find strange is that why don't we have a matching enum for something as basic as selling an item? Why is this value so vastly different? Is it bitwise OR'ed with something?

Reply:
I'm not sure i understand. enum values are added as they're found in the client code, if there's no entry yet for item sale, then it has to be added

Reply:
I mean, selling items is from start of the game. All the other enums extracted match up exact. I find it strange the numbers we have here are so much larger.

Reply:
Welp. Beats me. And the worst part is that the numbers for the same action change across protocol versions

Reply:
Are you sure the values you extracted are correct? Let's look at two of the values.

- ItemsBought = 0x10a
- UseJobItem = 0x121,

Let's look at the values in the enum
- ShopGoods_buy = 0xa,
- UseJobItem = 0x21,

Notice that each of these values are the same except for one extra bit added in the third byte.

Reply:
The values are taken straight from the decompilation when the item update wait flag is set. But chances are that maybe only the lowest byte is actually taken into account when comparing. You can try checking if the UI doesn't lock when the higher bytes are altered